packages/elements-core/src/utils/ref-resolving/resolvedObject.ts

```diff
@@ -78,7 +78,30 @@ export const isResolvedObjectProxy = (someObject: object): boolean => {
 };
 
 export const getOriginalObject = (resolvedObject: object): object => {
-  return (resolvedObject as Record<symbol, object>)[originalObjectSymbol] || resolvedObject;
+  const originalObject: any = (resolvedObject as Record<symbol, object>)[originalObjectSymbol] || resolvedObject;
```

Comment:
Is the function also handle the allOf usecase ?

Reply:
In allOf it is not showing empty object it is working as expected so just handled anyOf and oneOf.